libldbrest/batch.go

```diff
@@ -2,11 +2,14 @@ package libldbrest
 
 import (
 	"errors"
+
 	"github.com/syndtr/goleveldb/leveldb"
 )
 
 type oplist []*struct {
-	Op, Key, Value string
+	Op    string `json:"op"`
```

Comment:
I just remembered this:
We need to add the equivalent struct tag for msgpack, otherwise msgpack will serialize with key `Op` instead of `op`.

It should look like this, I believe:
```go
type oplist []*struct {
    Op string `json:"op" codec:"op"`
    ...
}
```